src/victus.js

```diff
@@ -281,13 +281,13 @@
 
       color = obj.color || "#fff";
     },
-    Rect: Rect,
-    Ellipse: Ellipse,
-    Sprite: Sprite,
-    Text: Text,
-    Sound: Sound,
+    Rect,
+    Ellipse,
+    Sprite,
+    Text,
+    Sound,
     clear: k,
```

Comment:
the shorthand is good! could you move line 289 so that it becomes the last item in the object, though?

Reply:
Of course!